src/main/java/racingcar/controller/RacingCarController.java

```diff
@@ -0,0 +1,16 @@
+package racingcar.controller;
+
+import racingcar.domain.RacingGame;
+import racingcar.domain.car.Cars;
+import racingcar.view.input.RacingCarParameter;
+
+import java.util.List;
+
+public class RacingCarController {
+
+    public List<Cars> getRacingCarResult(RacingCarParameter parameter) {
+        RacingGame racingGame = new RacingGame(parameter.getCarNames(), parameter.getRacingCount());
```

Comment:
```suggestion
        RacingGame racingGame = new RacingGame(parameter);
```
저는 개인적으로 파라미터가 늘어나는것을 방지하기 위해 DTO 객체만 전달하는 것을 선호한답니다 😄 
개인의 취향이니 이러한 방법도 사용된다는걸 알아주세요 !

Reply:
다양한 개발자의 취향도 반영하기 위해 생성자 추가하겠습니다 ㅎㅎ

Reply:
생성자가 많아지면 객체를 더욱 유연하게 사용할수 있어요. 
좋은 의견입니다 !